### PR TITLE
Support make install

### PR DESCRIPTION
**Pull request by seanshpark**
This will add installation to build folder that will
gather shared objects to same folder of game executable

### CODE REVIEW

**Comment by seanshpark** [comment]
For #35

**Comment by seanshpark** [comment]
To run `game`, need to cd to `game` folder, and this is not quite I wanted.
I've prepared a `play.py` that changes directory where `game` exist and executes it.

**Comment by wateret** [comment]
LGTM other than typo :)

File: game/CMakeLists.txt [hunk 1]
```diff
@@ -2,6 +2,13 @@ project(game)
 
 set(CMAKE_MACOSX_RPATH 1)
 
+set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+set(CMAKE_INSTALL_RPATH "./")
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+if(NOT INSTALL_FOLDER)
+  set(INSTALL_FOLDER, "menge")
```
[the remark this answers is not in the document]
Comment (seanshpark) [reply game/CMakeLists.txt]: Ah.. will fix :)

**Pull request merged**